generators/server/templates/src/main/resources/config/cql/_create-keyspace-prod.cql

```diff
@@ -1,3 +1,13 @@
 CREATE KEYSPACE IF NOT EXISTS <%= baseName %>
     WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 3 }
     AND DURABLE_WRITES = true;
+
+CREATE TABLE IF NOT EXISTS <%= baseName %>.schema_version (
```

Comment:
oh I understand why you move this table here! We need schema_version before scanning changelog/* and executing all cql files.
So forget my comment => but we need to find something to resolve unit tests